### ISSUE TITLE
Sanskrit Spanish Dictionaries

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/COLOGNE#305]
A recent email from Pas V.:

```
I'm from Chile, and I'm working on a project of sanskrit dictionaries in portuguese and spanish, 
and since you've been involved in the Cologne Digital Sanskrit Dictionaries, 
I was wondering if you could give some orientations and perhaps work in alliance. 
It would be nice to talk with you about this.
```

This seems an interesting idea.   Are there any existing Sanskrit-Spanish dictionaries ?

Perhaps Pas can elaborate on how he is thinking of developing this idea,  and perhaps others reading
this comment will have some ideas on the topic.

**Comment by gasyoun** [comment]
> Are there any existing Sanskrit-Spanish dictionaries ?

Yes, Oscar Pujol <oscar DOT pujol SIGN gmail.com> has made Sanskrit-Spanish based on MW after his Sanskrit-Catalan
https://www.youtube.com/watch?v=dOloOOo23jw I have some files from him, including list of headwords.

![catala](https://user-images.githubusercontent.com/80761/86590570-8aa19180-bf98-11ea-8c4e-402f9d3d5f3e.PNG)

![agni](https://user-images.githubusercontent.com/80761/86590576-8d9c8200-bf98-11ea-928d-225b8d15ff65.PNG)

**Comment by drdhaval2785** [comment]
A good idea would be to invite the interested scholar to github.

**Comment by gasyoun** [comment]
**Question type:** editorial-policy (whether to add Sanskrit–Spanish/Portuguese dictionaries to CDSL)

**Evidence found:**

No Sanskrit–Spanish or Sanskrit–Portuguese dictionary exists in csl-orig. The 45 dictionary directories in [csl-orig/v02/](https://github.com/sanskrit-lexicon/csl-orig/tree/main/v02) cover: German (PWG, PW, PWK, BUR), French (SCH), Russian (KOW), and English-language dictionaries (MW, MWE, AP90, AE, BEN, GRA, SHS, WIL, YAT, etc.) plus Sanskrit–Sanskrit (SKD, VCP). No `spa` or `por` directory exists.

The existing Sanskrit–Spanish dictionary is Oscar Pujol's work (mentioned in the first comment, 2020-07-06): derived from MW, with a prior Sanskrit–Catalan edition. @gasyoun confirmed having files from Pujol including a headword list. No further contact or collaboration is recorded in any open issue.

For comparison, the CDSL intake workflow for a new dictionary requires: a digitized source text in the csl-orig markup format, a pywork pipeline, and a display/DTB integration. The Pujol dictionary would need scanning/OCR or a digital source from the author.

**Assessment:** Answerable at the editorial-policy level — the question of whether Spanish dicts belong in CDSL is a `@DECIDE` item for the maintainers; technically there is no blocker if a digital source text exists. The factual question (does any Sanskrit-Spanish dict exist?) is answered: yes, Pujol's, but it is not digitized in CDSL format.

**Suggested next step:** Contact Oscar Pujol (oscar.pujol@gmail.com, mentioned in the 2020 comment) to ask if he can provide a digital source file in a parseable format. If yes, open a new issue to track the intake pipeline. Inviting the original emailer (Pas V.) to GitHub to share what format his Portuguese project data is in would also be useful.

**Comment by drdhaval2785** [comment]
@gasyoun may contact Pujol and Pas if thought fit.
